src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+li {
+  list-style: none;
+}
+
+.nav__link {
+  position: relative;
+  display: inline-block;
+  margin-left: 20px;
+  padding-top: 23px;
+  padding-bottom: 23px;
+  text-decoration: none;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  color: #000;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__link.is-active {
+  color: #00acdc;
+}
+
+.nav__link.is-active::after {
+  content: "";
+  position: absolute;
+  border-bottom: 4px solid #00acdc;
+  border-radius: 8px;
+  display: flex;
+  right: 0;
+  margin-top: 19px;
+  left: 0;
+  transition: all 1s;
+}
+
+.container {
+  display: flex;
+  max-width: calc(100% - 100px);
```

Comment:
just set paddings, I guess it will fix tests

Reply:
I can't figure out where to fix it.